subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/internal/DefaultJvmTestSuite.java

```diff
@@ -372,4 +374,12 @@ public void visitDependencies(TaskDependencyResolveContext context) {
             }
         };
     }
+
+    private static String getAppropriateSpockVersion() {
```

Comment:
In `JvmTestSuite`'s javadoc, we say the default version is 2.2-groovy-3.0.  Can we add a note here or figure out a way to not forget to change that when the time comes?